### ISSUE TITLE
nuScenes dataset testing

### ISSUE THREAD

**Issue by TJXA** [ucla-mobility/AutoVLA#42]
Great work! I’d like to ask a question regarding generalization. When testing on the nuScenes dataset, was the model fine-tuned before testing, or was the testing performed directly?

**Comment by Zewei-Zhou** [comment]
Hi,

The model was trained on a mix of nuPlan and nuScenes data, and then tested directly on nuScenes.

**Comment by TJXA** [comment]
Hello, I tested on the nuScenes dataset—specifically the validation set, which comprises 150 scenes and 5,569 data samples—but the results are unusual. The collision rate is very low, yet the L2 error is quite high.<img width="638" height="400" alt="Image" src="https://github.com/user-attachments/assets/25b375e6-2e84-488c-80c1-ee77a89cfc24" />

**Comment by Zewei-Zhou** [comment]
Hi,

Just to clarify, the checkpoints we released were trained with RFT on navsim. I meant that no additional SFT on nuScenes is required.

**Comment by TJXA** [comment]
Okay, thank you for the explanation.

**Comment by t-zhan** [comment]
> Hello, I tested on the nuScenes dataset—specifically the validation set, which comprises 150 scenes and 5,569 data samples—but the results are unusual. The collision rate is very low, yet the L2 error is quite high.<img alt="Image" width="638" height="400" src="https://private-user-images.githubusercontent.com/102894680/604023018-25b375e6-2e84-488c-80c1-ee77a89cfc24.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODI1Njc0MjIsIm5iZiI6MTc4MjU2NzEyMiwicGF0aCI6Ii8xMDI4OTQ2ODAvNjA0MDIzMDE4LTI1YjM3NWU2LTJlODQtNDg4Yy04MGMxLWVlNzdhODljZmMyNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwNjI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDYyN1QxMzMyMDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lYTY0YzQzZWNiNDA3YmZhOGQ3MjlkZTJhNjA2ZTc0M2Y5NzM1N2NmYWI2NzBlZjQxOWNiNWVlNmZlODVlMTgzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZyZXNwb25zZS1jb250ZW50LXR5cGU9aW1hZ2UlMkZwbmcifQ.5BjI11TPgLaDqjF7GYkPWloYzRPQV2hBsf0Sx12inbE">

相似的结果

<img width="1438" height="1228" alt="Image" src="https://github.com/user-attachments/assets/e0407996-419d-4e2e-868b-aca00d6aeedc" />

**Comment by ulairii** [comment]
I got similar results on the nuScenes dataset. It looks like the results are not aligned with the table in the appendix. 

<img width="1049" height="394" alt="Image" src="https://github.com/user-attachments/assets/f6a047a5-d35b-4ecc-9b06-8d36dc6fc576" />

**Comment by Zewei-Zhou** [comment]
Yes, you need to do the additional RFT on nuscenes, and the released checkpoint is based on navsim.
